### ISSUE TITLE
Feature request : Add shortcut to open terminal in the file's path

### ISSUE THREAD

**Issue by dglent** [tsujan/FeatherPad#441]
Add a shortcut (button , action) to open the terminal in the file's path in the editor

**Comment by tsujan** [comment]
Unlike the default file manager, there is no standard way to know about the "default terminal emulator". That's why it isn't added.

But it's possible to let the user define a terminal in Preferences. The item can be added to the tab/side-pane context menu, after "Open Containing Folder". I'll work on it after releasing 0.16.0.

**Comment by tsujan** [comment]
I thought about this. Although its implementation is easy, I'm afraid it isn't FeatherPad's job.

A text editor handles text files, not folders. Handling folders is the job of file managers, which should have a way of opening terminals inside folders (as Dolphin and PCManFM-Qt do).

Moreover, I've added "Open Containing Folder" to tab/item right-click menu. It calls the default file manager, which can open a terminal inside that folder and do more jobs. Calling a terminal by FeatherPad would be not only an overkill but also at odds with the file manager.

In short: Do one thing and do it well (Unix philosophy).
